pluto_build/bin/config.sh

```diff
@@ -43,6 +43,8 @@ function FGDB_export {
         -mapFieldType Integer64=Real\
         -lco GEOMETRY_NAME=Shape\
         -nln $@\
+        -nlt MULTIPOLYGON\
+        $@
```

Comment:
This last arg is the name of the table in the postgres database?

Reply:
yes

Reply:
yes